state/postgres_version.go

```diff
@@ -22,6 +22,7 @@ type PostgresVersion struct {
 	Numeric int    `json:"numeric"` // e.g. 90501
 
 	// For collector use only, to avoid calling functions that don't work
-	IsAwsAurora bool
-	IsCitus     bool
+	IsAwsAurora bool // Amazon Aurora
+	IsCitus     bool // Citus extension (e.g. with Azure CosmosDB for PostgreSQL)
+	IsEPAS      bool // EnterpriseDB Advanced Server
```

Comment:
FWIW, the reason this is not just "IsEDB" is because they actually have a few different options (e.g. "Extended Server"), but only Advanced Server is the one that has the additional catalog things (and lots of Oracle compatibility features), in my understanding.

The abbreviation "EPAS" is also used in their official docs, e.g. here: https://www.enterprisedb.com/docs/epas/latest/